pandablocks/commands.py

```diff
@@ -180,7 +190,7 @@ def execute(self) -> ExchangeGenerator[Union[str, List[str]]]:
         else:
             # We got OK =value
             line = ex.line
-            assert line.startswith("OK =")
+            assert line.startswith("OK ="), "Line did not start with 'OK ='"
```

Comment:
```suggestion
            assert line.startswith("OK ="), f"{line} does not start with 'OK ='"
```

Reply:
I added quotes around the `{line}`, as otherwise you end up with a slightly hard to read error message like this: `"AssertionError:ERR could not get description does not start with 'OK ='"`